### PR TITLE
Update some commands, mostly networking

### PR DESCRIPTION
**Pull request by kannibalox**

### CODE REVIEW

File: docs/include-cmd-network.rst [hunk 1]
```diff
@@ -13,13 +13,29 @@
     network.http.dns_cache_timeout
     network.http.dns_cache_timeout.set
 
-        **TODO**
+        .. code-block:: ini
+
+           network.http.max_open.set = ‹seconds› ≫ 0
+           network.http.max_open ≫ ‹seconds›
```
Comment (pyroscope) [inline docs/include-cmd-network.rst]: cmd name copy+paste error

File: docs/include-cmd-items.rst [hunk 2]
```diff
@@ -110,6 +124,9 @@
     d.ratio
     d.resume
```
Comment (pyroscope) [inline docs/include-cmd-items.rst:128]: Add a new **TODO** (or move the existing one) if neighboring commands are not related to your insertion.

File: docs/include-cmd-items.rst [hunk 3]
```diff
@@ -3,6 +3,12 @@
 `d.*` commands
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+All `d.*` commands take a torrent hash as the first argument when called over the XMLRPC API.
+
```
Comment (pyroscope) [inline docs/include-cmd-items.rst]: "info hash"

**Review by pyroscope — CHANGES_REQUESTED** [review]
some fixes needed.

**Comment by pyroscope** [comment]
:+1: Great! If more people invest some time, this might actually get complete.

**Comment by kannibalox** [comment]
I've updated the PR, I'll keep it them limited to one file in the future too.

**Review by pyroscope — APPROVED** [review]

**Pull request merged**